js/data/bgutil.js

```diff
@@ -156,12 +156,12 @@ function BGUtil(data, opts) {
       }, 0);
       var average;
       if (opts.bgUnits === 'mmol/L') {
```

Comment:
Ok, I've gone through the app and changed all occurrences to use constants.

About 30 files in total needed updating.

I've done a quick smoke test, and everything looks good to me, but it may be good to have you quickly scan it as well in case I missed something.